### PR TITLE
edit to ITP prep git

### PR DESCRIPTION
**Pull request by fcaroline2020**
## What does this change?

adds that git will be covered more indepth later and includes installation instructions

### Common Content?

No

### Common Theme?

Issue number: #782 

### Org Content?

Sprint

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

**Review by moneyinthesky — APPROVED** [review]

File: common-content/en/module/induction/check-git-installation/index.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ emoji= '🧩'
 
 ### Git
 
-You will use Git continually as a developer.
+You will use Git continually as a developer. We will cover Git more in depth later in the course, right now, we will just check that you have it installed.
```
Comment (SallyMcGrath) [inline common-content/en/module/induction/check-git-installation/index.md:14]:
```suggestion
You will use Git continually as a developer. We will cover Git in more depth later in the course. Right now, we will just check that you have it installed.
```

Comment (SallyMcGrath) [reply common-content/en/module/induction/check-git-installation/index.md:14]: Comma splice

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
When changing times for a day plan, change them on the day plan, not the central block.

**Comment by JayMayer** [comment]
@fcaroline2020 With Sally's suggestions, looks good to me. Can they be accepted to get this one merged?

**Review by SallyMcGrath — APPROVED** [review]

**Pull request merged**